### PR TITLE
Fix up k8s.gcr.io freeze blog article

### PR DESCRIPTION
**Pull request by upodroid**
/cc @sftim
/label tide/merge-method-squash

### CODE REVIEW

File: content/en/blog/_posts/2023-02-06-k8s-gcr-io-freeze-announcement.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: blog
 title: "k8s.gcr.io image registry will be frozen from the 3rd of April 2023"
-date: 2023-02-06
+date: 2023-02-06T17:00:00
 slug: k8s-gcr-io-freeze-announcement
```
Comment (reylejano) [inline content/en/blog/_posts/2023-02-06-k8s-gcr-io-freeze-announcement.md:5]: nit: Can you make the title sentence case?
title: "k8s.gcr.io Image Registry Will Be Frozen From the 3rd of April 2023"

**Comment by sftim** [comment]
/retitle Fix up k8s.gcr.io freeze blog article

File: content/en/blog/_posts/2023-02-06-k8s-gcr-io-freeze-announcement.md [hunk 2]
```diff
@@ -20,11 +20,11 @@ What does this change mean for end users:
 - Starting in 1.25, the default image registry has been set to `registry.k8s.io`. This value is overridable in `kubeadm` and `kubelet` but setting it to `k8s.gcr.io` will fail for new releases after April as they won’t be present in the old registry.
 - If you want to increase the reliability of your cluster and remove dependency on the community-owned registry or you are running Kubernetes in networks where external traffic is restricted, you should consider hosting local image registry mirrors. Some cloud vendors may offer hosted solutions for this.
 
-## Timeline of the Changes:
+## Timeline of the changes:
```
Comment (sftim) [inline content/en/blog/_posts/2023-02-06-k8s-gcr-io-freeze-announcement.md:23]:
```suggestion
## Timeline of the changes
```

**Review by sftim — COMMENTED** [review]
/lgtm

@upodroid this will be tidier if you accept feedback, pull, squash locally, and then force push back. However you don't have to do that; Tide can do a less tidy squash merge instead (and that's configured right now).

File: content/en/blog/_posts/2023-02-06-k8s-gcr-io-freeze-announcement.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: blog
-title: "k8s.gcr.io image registry will be frozen from the 3rd of April 2023"
-date: 2023-02-06
+title: "k8s.gcr.io Image Registry Will Be Frozen From the 3rd of April 2023"
+date: 2023-02-06T17:00:00
```
Comment (sftim) [inline content/en/blog/_posts/2023-02-06-k8s-gcr-io-freeze-announcement.md:4]: Now that the article is published, I'm wary of changing the publication date.

**Comment by sftim** [comment]
/lgtm cancel

I now don't want to update the publication date / time stamp - for some readers, it could show as a new article in feeds.

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm
/approve

**Pull request merged**